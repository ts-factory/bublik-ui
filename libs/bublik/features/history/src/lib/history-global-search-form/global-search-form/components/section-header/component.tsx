/* SPDX-License-Identifier: Apache-2.0 */
/* SPDX-FileCopyrightText: 2021-2023 OKTET Labs Ltd. */
import { ComponentPropsWithRef, FC, forwardRef } from 'react';

export interface FormSectionHeaderProps extends ComponentPropsWithRef<'div'> {
	name: string;
	error?: string;
}

export const FormSectionHeader: FC<FormSectionHeaderProps> = forwardRef(
	({ name, error, children, ...props }, ref) => {
		return (
			<div className="mb-4" {...props}>
				<div className="flex items-center justify-between" ref={ref}>
					<legend className="text-lg font-semibold leading-[0.875rem]">
						{name}
					</legend>
					<div className="flex items-center gap-2">{children}</div>
				</div>
				{error && (
					<span className="text-[0.75rem] leading-[0.875rem] text-text-unexpected">
						{error}
					</span>
				)}
			</div>
		);
	}
);

FormSectionHeader.displayName = 'FormSectionHeader';
