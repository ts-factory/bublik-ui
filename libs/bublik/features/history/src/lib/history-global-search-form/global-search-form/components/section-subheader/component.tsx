/* SPDX-License-Identifier: Apache-2.0 */
/* SPDX-FileCopyrightText: 2021-2023 OKTET Labs Ltd. */
import { FC } from 'react';

export interface FormSectionSubheaderProps {
	name: string;
}

export const FormSectionSubheader: FC<FormSectionSubheaderProps> = ({
	name
}) => {
	return (
		<div className="">
			<span className="text-text-secondary text-sm font-semibold">{name}</span>
		</div>
	);
};
