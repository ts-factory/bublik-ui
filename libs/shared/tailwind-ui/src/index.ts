/* SPDX-License-Identifier: Apache-2.0 */
/* SPDX-FileCopyrightText: 2021-2023 OKTET Labs Ltd. */
export * from './lib/badge-input';
export * from './lib/checkbox';
export * from './lib/radio';
export * from './lib/input';
export * from './lib/badge';
export * from './lib/info-card';
export * from './lib/pagination';
export * from './lib/spinner';
export * from './lib/badges-header-input';
export * from './lib/scroll-to-top';
export * from './lib/tooltip';
export * from './lib/verdict-list';
export * from './lib/badge-list';
export * from './lib/copy-tooltip';
export * from './lib/error-boundary';
export * from './lib/iframe-to-old';
export * from './lib/badge-input';
export * from './lib/scroll-area';
export * from './lib/env-badge';
export * from './lib/table-sort';
export * from './lib/combo-select';
export * from './lib/tw-table';
export * from './lib/button';
export * from './lib/app-shell';
export * from './lib/utils';
export * from './lib/run-mode-toggle';
export * from './lib/clock';
export * from './lib/not-found';
export * from './lib/card-header';
export * from './lib/search-bar';
export * from './lib/popover';
export * from './lib/dialog';
export * from './lib/connection-status';
export * from './lib/resizable';
export * from './lib/toggle-switch';
export * from './lib/providers';
export * from './lib/side-panel';
export * from './lib/error-page';
export * from './lib/search-autocomplete';
export * from './lib/storybook-bg';
export * from './lib/switch-date-picker';
export * from './lib/hover-card';
export * from './lib/table-tree-node';
export * from './lib/toggle-group';
export * from './lib/context-menu';
export * from './lib/styles';
export * from './lib/text-area';
export * from './lib/json-viewer';
export * from './lib/skeleton';
export * from './lib/dropdown';
export * from './lib/collapsible';
export * from './lib/icon';
export * from './lib/command';
export * from './lib/separator';
export * from './lib/badge-box';
export * from './lib/toolbar';
export * from './lib/drawer';
export * from './lib/context-links';
export * from './lib/form-alert';
export * from './lib/alert-dialog';
export * from './lib/avatar';
export * from './lib/sonner-toaster';
export * from './lib/radio-group';
export * from './lib/conclusion-hover-card';
export * from './lib/conclusion-badge';
export * from './lib/instruction-dialog';
export * from './lib/data-table-faceted-filter';
export * from './lib/run-running';
export * from './lib/split-button';
