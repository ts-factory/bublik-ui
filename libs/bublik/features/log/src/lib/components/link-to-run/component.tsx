/* SPDX-License-Identifier: Apache-2.0 */
/* SPDX-FileCopyrightText: 2021-2023 OKTET Labs Ltd. */
import { ButtonTw, Icon } from '@/shared/tailwind-ui';
import { LinkWithProject } from '@/bublik/features/projects';

export interface LinkToRunProps {
	runId: string;
}

export function LinkToRun({ runId }: LinkToRunProps) {
	const to = { pathname: `/runs/${runId}` };

	return (
		<ButtonTw asChild variant="secondary" size="xss">
			<LinkWithProject to={to}>
				<Icon name="BoxArrowRight" className="mr-1.5" />
				Run
			</LinkWithProject>
		</ButtonTw>
	);
}
