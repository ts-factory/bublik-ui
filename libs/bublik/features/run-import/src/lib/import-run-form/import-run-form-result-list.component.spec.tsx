/* SPDX-License-Identifier: Apache-2.0 */
/* SPDX-FileCopyrightText: 2021-2023 OKTET Labs Ltd. */
import { Provider } from 'react-redux';
import { configureStore } from '@reduxjs/toolkit';
import { Dialog, DialogContent } from '@/shared/tailwind-ui';
import { render } from '@testing-library/react';

import { RunImportResult } from './import-run-form-result-list.component';

describe('ImportRunFormResultList', () => {
	it('should render successfully', () => {
		const { asFragment } = render(
			<Provider store={configureStore({ reducer: {} })}>
				<Dialog open>
					<DialogContent>
						<RunImportResult
							results={[
								{ url: 'https://www.youtube.com/watch?v=1', taskId: '1' },
								{ url: 'https://www.youtube.com/watch?v=2', taskId: '2' },
								{ url: 'https://www.youtube.com/watch?v=3', taskId: '3' }
							]}
						/>
					</DialogContent>
				</Dialog>
			</Provider>
		);

		expect(asFragment()).toMatchSnapshot();
	});
});
