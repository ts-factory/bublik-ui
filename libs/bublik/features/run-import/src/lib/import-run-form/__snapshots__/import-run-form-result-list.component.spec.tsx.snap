// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ImportRunFormResultList > should render successfully 1`] = `
<DocumentFragment>
  <div
    aria-describedby="radix-:r2:"
    aria-labelledby="radix-:r1:"
    data-state="open"
    id="radix-:r0:"
    role="dialog"
    style="pointer-events: auto;"
    tabindex="-1"
  >
    <div>
      <h2
        class="text-lg font-medium leading-6 text-gray-900"
        id="radix-:r1:"
      >
        Import runs
      </h2>
      <p
        class="mt-2 text-sm font-normal text-gray-700"
        id="radix-:r2:"
      >
        Scheduled runs will be imported in the background. 
        <br />
         You can check the logs and flower tasks
      </p>
      <ul
        class="flex flex-col gap-2 mt-2 divide-y divide-border-primary"
      >
        <li>
          <div
            class="flex items-center gap-4 p-2"
          >
            <div
              class="flex flex-col w-24 gap-1"
            >
              <button
                aria-controls="radix-:r3:"
                aria-expanded="false"
                aria-haspopup="dialog"
                class="relative inline-flex items-center justify-start px-2 w-fit transition-all appearance-none select-none whitespace-nowrap text-primary bg-primary-wash rounded-md gap-1 h-[1.625rem] border-2 border-transparent hover:border-[#94b0ff]"
                data-state="closed"
                type="button"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M2.3065 7.50967C1.95484 7.50967 1.669 7.22384 1.669 6.87217C1.6665 3.65967 3.65984 1.66634 6.86984 1.66634H13.1223C16.3398 1.66634 18.3332 3.65967 18.3332 6.86967V13.1197C18.3332 16.3388 16.3398 18.333 13.1223 18.333H6.87234C3.67484 18.333 1.6665 16.3388 1.6665 13.1222L1.6665 10.9813C1.6665 10.6338 1.949 10.3513 2.2965 10.3513H2.304V10.3655C2.65567 10.3655 2.94067 10.6505 2.9415 11.0022V11.0038L2.9415 13.1222C2.9415 15.6663 4.33317 17.058 6.86984 17.058H13.1198C15.6582 17.058 17.0582 15.658 17.0582 13.1222V6.87217C17.0582 4.34384 15.6582 2.94384 13.1223 2.94384H6.87234C4.3365 2.94384 2.944 4.34384 2.944 6.87217C2.944 7.22384 2.65817 7.50967 2.3065 7.50967ZM6.59492 10.6246H11.8916L9.82742 12.6804C9.58242 12.9238 9.58159 13.3188 9.82575 13.5638C10.0691 13.8096 10.4649 13.8079 10.7091 13.5654L13.8458 10.4421C13.8466 10.4421 13.8466 10.4421 13.8466 10.4413C13.9041 10.3846 13.9499 10.3154 13.9816 10.2388C14.0133 10.1629 14.0299 10.0821 14.0299 10.0004V9.99959C14.0299 9.99377 14.0285 9.98891 14.027 9.98398C14.0258 9.97973 14.0245 9.97543 14.0241 9.97042C14.0208 9.89959 14.0091 9.82876 13.9816 9.76126C13.9533 9.69292 13.9108 9.63292 13.8616 9.57792C13.8589 9.57522 13.8571 9.57217 13.8553 9.56909C13.8531 9.56551 13.851 9.56189 13.8474 9.55876C13.7899 9.50042 13.7208 9.45459 13.6449 9.42292C13.5674 9.39126 13.4874 9.37459 13.4049 9.37459H6.59492C6.24992 9.37459 5.96992 9.65459 5.96992 9.99959C5.96992 10.3446 6.24992 10.6246 6.59492 10.6246ZM11.7797 7.49876L10.7097 6.43376C10.5872 6.31209 10.428 6.25126 10.2688 6.25126C10.108 6.25126 9.948 6.31209 9.82467 6.43542C9.58134 6.68042 9.58217 7.07542 9.82717 7.31876L10.8972 8.38376C11.1422 8.62709 11.5372 8.62709 11.7813 8.38292C12.0247 8.13792 12.0238 7.74209 11.7797 7.49876Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
                <span>
                  Logs
                </span>
              </button>
              <a
                class="relative inline-flex items-center justify-start px-2 w-fit transition-all appearance-none select-none whitespace-nowrap text-primary bg-primary-wash rounded-md gap-1 h-[1.625rem] border-2 border-transparent hover:border-[#94b0ff] text-xs"
                href="http://localhost:3000/flower/task/1"
                rel="noreferrer"
                target="_blank"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M2.3065 7.50967C1.95484 7.50967 1.669 7.22384 1.669 6.87217C1.6665 3.65967 3.65984 1.66634 6.86984 1.66634H13.1223C16.3398 1.66634 18.3332 3.65967 18.3332 6.86967V13.1197C18.3332 16.3388 16.3398 18.333 13.1223 18.333H6.87234C3.67484 18.333 1.6665 16.3388 1.6665 13.1222L1.6665 10.9813C1.6665 10.6338 1.949 10.3513 2.2965 10.3513H2.304V10.3655C2.65567 10.3655 2.94067 10.6505 2.9415 11.0022V11.0038L2.9415 13.1222C2.9415 15.6663 4.33317 17.058 6.86984 17.058H13.1198C15.6582 17.058 17.0582 15.658 17.0582 13.1222V6.87217C17.0582 4.34384 15.6582 2.94384 13.1223 2.94384H6.87234C4.3365 2.94384 2.944 4.34384 2.944 6.87217C2.944 7.22384 2.65817 7.50967 2.3065 7.50967ZM6.59492 10.6246H11.8916L9.82742 12.6804C9.58242 12.9238 9.58159 13.3188 9.82575 13.5638C10.0691 13.8096 10.4649 13.8079 10.7091 13.5654L13.8458 10.4421C13.8466 10.4421 13.8466 10.4421 13.8466 10.4413C13.9041 10.3846 13.9499 10.3154 13.9816 10.2388C14.0133 10.1629 14.0299 10.0821 14.0299 10.0004V9.99959C14.0299 9.99377 14.0285 9.98891 14.027 9.98398C14.0258 9.97973 14.0245 9.97543 14.0241 9.97042C14.0208 9.89959 14.0091 9.82876 13.9816 9.76126C13.9533 9.69292 13.9108 9.63292 13.8616 9.57792C13.8589 9.57522 13.8571 9.57217 13.8553 9.56909C13.8531 9.56551 13.851 9.56189 13.8474 9.55876C13.7899 9.50042 13.7208 9.45459 13.6449 9.42292C13.5674 9.39126 13.4874 9.37459 13.4049 9.37459H6.59492C6.24992 9.37459 5.96992 9.65459 5.96992 9.99959C5.96992 10.3446 6.24992 10.6246 6.59492 10.6246ZM11.7797 7.49876L10.7097 6.43376C10.5872 6.31209 10.428 6.25126 10.2688 6.25126C10.108 6.25126 9.948 6.31209 9.82467 6.43542C9.58134 6.68042 9.58217 7.07542 9.82717 7.31876L10.8972 8.38376C11.1422 8.62709 11.5372 8.62709 11.7813 8.38292C12.0247 8.13792 12.0238 7.74209 11.7797 7.49876Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
                <span>
                  Flower
                </span>
              </a>
            </div>
            <div
              class="w-24"
            >
              <div
                class="inline-flex px-2 text-xs font-semibold leading-5 rounded items-center bg-badge-3 text-text-expected"
              >
                STARTED
              </div>
            </div>
            <div
              class="flex-grow"
            >
              <a
                class="text-sm font-normal hover:underline"
                href="https://www.youtube.com/watch?v=1"
                rel="noreferrer"
                target="_blank"
              >
                https://www.youtube.com/watch?v=1
              </a>
            </div>
          </div>
        </li>
        <li>
          <div
            class="flex items-center gap-4 p-2"
          >
            <div
              class="flex flex-col w-24 gap-1"
            >
              <button
                aria-controls="radix-:r6:"
                aria-expanded="false"
                aria-haspopup="dialog"
                class="relative inline-flex items-center justify-start px-2 w-fit transition-all appearance-none select-none whitespace-nowrap text-primary bg-primary-wash rounded-md gap-1 h-[1.625rem] border-2 border-transparent hover:border-[#94b0ff]"
                data-state="closed"
                type="button"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M2.3065 7.50967C1.95484 7.50967 1.669 7.22384 1.669 6.87217C1.6665 3.65967 3.65984 1.66634 6.86984 1.66634H13.1223C16.3398 1.66634 18.3332 3.65967 18.3332 6.86967V13.1197C18.3332 16.3388 16.3398 18.333 13.1223 18.333H6.87234C3.67484 18.333 1.6665 16.3388 1.6665 13.1222L1.6665 10.9813C1.6665 10.6338 1.949 10.3513 2.2965 10.3513H2.304V10.3655C2.65567 10.3655 2.94067 10.6505 2.9415 11.0022V11.0038L2.9415 13.1222C2.9415 15.6663 4.33317 17.058 6.86984 17.058H13.1198C15.6582 17.058 17.0582 15.658 17.0582 13.1222V6.87217C17.0582 4.34384 15.6582 2.94384 13.1223 2.94384H6.87234C4.3365 2.94384 2.944 4.34384 2.944 6.87217C2.944 7.22384 2.65817 7.50967 2.3065 7.50967ZM6.59492 10.6246H11.8916L9.82742 12.6804C9.58242 12.9238 9.58159 13.3188 9.82575 13.5638C10.0691 13.8096 10.4649 13.8079 10.7091 13.5654L13.8458 10.4421C13.8466 10.4421 13.8466 10.4421 13.8466 10.4413C13.9041 10.3846 13.9499 10.3154 13.9816 10.2388C14.0133 10.1629 14.0299 10.0821 14.0299 10.0004V9.99959C14.0299 9.99377 14.0285 9.98891 14.027 9.98398C14.0258 9.97973 14.0245 9.97543 14.0241 9.97042C14.0208 9.89959 14.0091 9.82876 13.9816 9.76126C13.9533 9.69292 13.9108 9.63292 13.8616 9.57792C13.8589 9.57522 13.8571 9.57217 13.8553 9.56909C13.8531 9.56551 13.851 9.56189 13.8474 9.55876C13.7899 9.50042 13.7208 9.45459 13.6449 9.42292C13.5674 9.39126 13.4874 9.37459 13.4049 9.37459H6.59492C6.24992 9.37459 5.96992 9.65459 5.96992 9.99959C5.96992 10.3446 6.24992 10.6246 6.59492 10.6246ZM11.7797 7.49876L10.7097 6.43376C10.5872 6.31209 10.428 6.25126 10.2688 6.25126C10.108 6.25126 9.948 6.31209 9.82467 6.43542C9.58134 6.68042 9.58217 7.07542 9.82717 7.31876L10.8972 8.38376C11.1422 8.62709 11.5372 8.62709 11.7813 8.38292C12.0247 8.13792 12.0238 7.74209 11.7797 7.49876Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
                <span>
                  Logs
                </span>
              </button>
              <a
                class="relative inline-flex items-center justify-start px-2 w-fit transition-all appearance-none select-none whitespace-nowrap text-primary bg-primary-wash rounded-md gap-1 h-[1.625rem] border-2 border-transparent hover:border-[#94b0ff] text-xs"
                href="http://localhost:3000/flower/task/2"
                rel="noreferrer"
                target="_blank"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M2.3065 7.50967C1.95484 7.50967 1.669 7.22384 1.669 6.87217C1.6665 3.65967 3.65984 1.66634 6.86984 1.66634H13.1223C16.3398 1.66634 18.3332 3.65967 18.3332 6.86967V13.1197C18.3332 16.3388 16.3398 18.333 13.1223 18.333H6.87234C3.67484 18.333 1.6665 16.3388 1.6665 13.1222L1.6665 10.9813C1.6665 10.6338 1.949 10.3513 2.2965 10.3513H2.304V10.3655C2.65567 10.3655 2.94067 10.6505 2.9415 11.0022V11.0038L2.9415 13.1222C2.9415 15.6663 4.33317 17.058 6.86984 17.058H13.1198C15.6582 17.058 17.0582 15.658 17.0582 13.1222V6.87217C17.0582 4.34384 15.6582 2.94384 13.1223 2.94384H6.87234C4.3365 2.94384 2.944 4.34384 2.944 6.87217C2.944 7.22384 2.65817 7.50967 2.3065 7.50967ZM6.59492 10.6246H11.8916L9.82742 12.6804C9.58242 12.9238 9.58159 13.3188 9.82575 13.5638C10.0691 13.8096 10.4649 13.8079 10.7091 13.5654L13.8458 10.4421C13.8466 10.4421 13.8466 10.4421 13.8466 10.4413C13.9041 10.3846 13.9499 10.3154 13.9816 10.2388C14.0133 10.1629 14.0299 10.0821 14.0299 10.0004V9.99959C14.0299 9.99377 14.0285 9.98891 14.027 9.98398C14.0258 9.97973 14.0245 9.97543 14.0241 9.97042C14.0208 9.89959 14.0091 9.82876 13.9816 9.76126C13.9533 9.69292 13.9108 9.63292 13.8616 9.57792C13.8589 9.57522 13.8571 9.57217 13.8553 9.56909C13.8531 9.56551 13.851 9.56189 13.8474 9.55876C13.7899 9.50042 13.7208 9.45459 13.6449 9.42292C13.5674 9.39126 13.4874 9.37459 13.4049 9.37459H6.59492C6.24992 9.37459 5.96992 9.65459 5.96992 9.99959C5.96992 10.3446 6.24992 10.6246 6.59492 10.6246ZM11.7797 7.49876L10.7097 6.43376C10.5872 6.31209 10.428 6.25126 10.2688 6.25126C10.108 6.25126 9.948 6.31209 9.82467 6.43542C9.58134 6.68042 9.58217 7.07542 9.82717 7.31876L10.8972 8.38376C11.1422 8.62709 11.5372 8.62709 11.7813 8.38292C12.0247 8.13792 12.0238 7.74209 11.7797 7.49876Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
                <span>
                  Flower
                </span>
              </a>
            </div>
            <div
              class="w-24"
            >
              <div
                class="inline-flex px-2 text-xs font-semibold leading-5 rounded items-center bg-badge-3 text-text-expected"
              >
                STARTED
              </div>
            </div>
            <div
              class="flex-grow"
            >
              <a
                class="text-sm font-normal hover:underline"
                href="https://www.youtube.com/watch?v=2"
                rel="noreferrer"
                target="_blank"
              >
                https://www.youtube.com/watch?v=2
              </a>
            </div>
          </div>
        </li>
        <li>
          <div
            class="flex items-center gap-4 p-2"
          >
            <div
              class="flex flex-col w-24 gap-1"
            >
              <button
                aria-controls="radix-:r9:"
                aria-expanded="false"
                aria-haspopup="dialog"
                class="relative inline-flex items-center justify-start px-2 w-fit transition-all appearance-none select-none whitespace-nowrap text-primary bg-primary-wash rounded-md gap-1 h-[1.625rem] border-2 border-transparent hover:border-[#94b0ff]"
                data-state="closed"
                type="button"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M2.3065 7.50967C1.95484 7.50967 1.669 7.22384 1.669 6.87217C1.6665 3.65967 3.65984 1.66634 6.86984 1.66634H13.1223C16.3398 1.66634 18.3332 3.65967 18.3332 6.86967V13.1197C18.3332 16.3388 16.3398 18.333 13.1223 18.333H6.87234C3.67484 18.333 1.6665 16.3388 1.6665 13.1222L1.6665 10.9813C1.6665 10.6338 1.949 10.3513 2.2965 10.3513H2.304V10.3655C2.65567 10.3655 2.94067 10.6505 2.9415 11.0022V11.0038L2.9415 13.1222C2.9415 15.6663 4.33317 17.058 6.86984 17.058H13.1198C15.6582 17.058 17.0582 15.658 17.0582 13.1222V6.87217C17.0582 4.34384 15.6582 2.94384 13.1223 2.94384H6.87234C4.3365 2.94384 2.944 4.34384 2.944 6.87217C2.944 7.22384 2.65817 7.50967 2.3065 7.50967ZM6.59492 10.6246H11.8916L9.82742 12.6804C9.58242 12.9238 9.58159 13.3188 9.82575 13.5638C10.0691 13.8096 10.4649 13.8079 10.7091 13.5654L13.8458 10.4421C13.8466 10.4421 13.8466 10.4421 13.8466 10.4413C13.9041 10.3846 13.9499 10.3154 13.9816 10.2388C14.0133 10.1629 14.0299 10.0821 14.0299 10.0004V9.99959C14.0299 9.99377 14.0285 9.98891 14.027 9.98398C14.0258 9.97973 14.0245 9.97543 14.0241 9.97042C14.0208 9.89959 14.0091 9.82876 13.9816 9.76126C13.9533 9.69292 13.9108 9.63292 13.8616 9.57792C13.8589 9.57522 13.8571 9.57217 13.8553 9.56909C13.8531 9.56551 13.851 9.56189 13.8474 9.55876C13.7899 9.50042 13.7208 9.45459 13.6449 9.42292C13.5674 9.39126 13.4874 9.37459 13.4049 9.37459H6.59492C6.24992 9.37459 5.96992 9.65459 5.96992 9.99959C5.96992 10.3446 6.24992 10.6246 6.59492 10.6246ZM11.7797 7.49876L10.7097 6.43376C10.5872 6.31209 10.428 6.25126 10.2688 6.25126C10.108 6.25126 9.948 6.31209 9.82467 6.43542C9.58134 6.68042 9.58217 7.07542 9.82717 7.31876L10.8972 8.38376C11.1422 8.62709 11.5372 8.62709 11.7813 8.38292C12.0247 8.13792 12.0238 7.74209 11.7797 7.49876Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
                <span>
                  Logs
                </span>
              </button>
              <a
                class="relative inline-flex items-center justify-start px-2 w-fit transition-all appearance-none select-none whitespace-nowrap text-primary bg-primary-wash rounded-md gap-1 h-[1.625rem] border-2 border-transparent hover:border-[#94b0ff] text-xs"
                href="http://localhost:3000/flower/task/3"
                rel="noreferrer"
                target="_blank"
              >
                <svg
                  fill="none"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M2.3065 7.50967C1.95484 7.50967 1.669 7.22384 1.669 6.87217C1.6665 3.65967 3.65984 1.66634 6.86984 1.66634H13.1223C16.3398 1.66634 18.3332 3.65967 18.3332 6.86967V13.1197C18.3332 16.3388 16.3398 18.333 13.1223 18.333H6.87234C3.67484 18.333 1.6665 16.3388 1.6665 13.1222L1.6665 10.9813C1.6665 10.6338 1.949 10.3513 2.2965 10.3513H2.304V10.3655C2.65567 10.3655 2.94067 10.6505 2.9415 11.0022V11.0038L2.9415 13.1222C2.9415 15.6663 4.33317 17.058 6.86984 17.058H13.1198C15.6582 17.058 17.0582 15.658 17.0582 13.1222V6.87217C17.0582 4.34384 15.6582 2.94384 13.1223 2.94384H6.87234C4.3365 2.94384 2.944 4.34384 2.944 6.87217C2.944 7.22384 2.65817 7.50967 2.3065 7.50967ZM6.59492 10.6246H11.8916L9.82742 12.6804C9.58242 12.9238 9.58159 13.3188 9.82575 13.5638C10.0691 13.8096 10.4649 13.8079 10.7091 13.5654L13.8458 10.4421C13.8466 10.4421 13.8466 10.4421 13.8466 10.4413C13.9041 10.3846 13.9499 10.3154 13.9816 10.2388C14.0133 10.1629 14.0299 10.0821 14.0299 10.0004V9.99959C14.0299 9.99377 14.0285 9.98891 14.027 9.98398C14.0258 9.97973 14.0245 9.97543 14.0241 9.97042C14.0208 9.89959 14.0091 9.82876 13.9816 9.76126C13.9533 9.69292 13.9108 9.63292 13.8616 9.57792C13.8589 9.57522 13.8571 9.57217 13.8553 9.56909C13.8531 9.56551 13.851 9.56189 13.8474 9.55876C13.7899 9.50042 13.7208 9.45459 13.6449 9.42292C13.5674 9.39126 13.4874 9.37459 13.4049 9.37459H6.59492C6.24992 9.37459 5.96992 9.65459 5.96992 9.99959C5.96992 10.3446 6.24992 10.6246 6.59492 10.6246ZM11.7797 7.49876L10.7097 6.43376C10.5872 6.31209 10.428 6.25126 10.2688 6.25126C10.108 6.25126 9.948 6.31209 9.82467 6.43542C9.58134 6.68042 9.58217 7.07542 9.82717 7.31876L10.8972 8.38376C11.1422 8.62709 11.5372 8.62709 11.7813 8.38292C12.0247 8.13792 12.0238 7.74209 11.7797 7.49876Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
                <span>
                  Flower
                </span>
              </a>
            </div>
            <div
              class="w-24"
            >
              <div
                class="inline-flex px-2 text-xs font-semibold leading-5 rounded items-center bg-badge-3 text-text-expected"
              >
                STARTED
              </div>
            </div>
            <div
              class="flex-grow"
            >
              <a
                class="text-sm font-normal hover:underline"
                href="https://www.youtube.com/watch?v=3"
                rel="noreferrer"
                target="_blank"
              >
                https://www.youtube.com/watch?v=3
              </a>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
